vendor/bat-native-ledger/src/bat/ledger/internal/promotion/promotion.cc

```diff
@@ -480,7 +480,7 @@ void Promotion::ClaimTokens(
   for (auto & token : tokens) {
     auto token_base64 = token.encode_base64();
     auto token_value = base::Value(token_base64);
-    tokens_list.GetList().push_back(std::move(token_value));
+    tokens_list.Insert(tokens_list.GetList().end(), std::move(token_value));
```

Comment:
what if we just do `tokens_list.Append(std::move(token_value));`?

Reply:
same for all other push_back where we have value list

Reply:
Yes! Definitely a much nicer way. Didn't notice `Append` method. Will update all the places.